### ISSUE TITLE
module 'logging' has no attribute 'getLogger'

### ISSUE THREAD

**Issue by rldhont** [3liz/lizmap-plugin#348]
## Description
```
16:20:50 WARNING Python error[6466]: Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/qgis/utils.py", line 590, in startServerPlugin
    server_plugins[packageName] = package.serverClassFactory(serverIface)
  File "/srv/qgis/plugins/lizmap/__init__.py", line 69, in serverClassFactory
    from lizmap.server.lizmap_server import LizmapServer
  File "/usr/lib/python3/dist-packages/qgis/utils.py", line 685, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "/srv/qgis/plugins/lizmap/server/lizmap_server.py", line 10, in <module>
    from .get_feature_info import GetFeatureInfoFilter
  File "/usr/lib/python3/dist-packages/qgis/utils.py", line 685, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "/srv/qgis/plugins/lizmap/server/get_feature_info.py", line 26, in <module>
    from lizmap.tooltip import Tooltip
  File "/usr/lib/python3/dist-packages/qgis/utils.py", line 685, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "/srv/qgis/plugins/lizmap/tooltip.py", line 16, in <module>
    LOGGER = logging.getLogger('Lizmap')
AttributeError: module 'logging' has no attribute 'getLogger'

16:20:50 WARNING Couldn't load server plugin lizmap due to an error when calling its serverClassFactory() method[6466]: Couldn't load server plugin lizmap due to an error when calling its serverClassFactory() method
```

## Context
* QGIS version: 3.4.15
* Lizmap version: 3.4.4
* OS: Debian 9

**Comment by dmarteau** [comment]
The module https://github.com/3liz/lizmap-plugin/blob/master/lizmap/server/logging.py hide the python library package

**Comment by Gustry** [comment]
Fixed in e8f31ecaeb63ef7c2e5c0ff2891f6ef029cea94a
